lib/lutaml/xml/parsers/xml.rb

```diff
@@ -0,0 +1,57 @@
+require "nokogiri"
+require "htmlentities"
+require "lutaml/uml/has_attributes"
+require "lutaml/uml/document"
+
+require "shale/schema"
+
+module Lutaml
+  module Xml
+    module Parsers
+      # Class for parsing .xml schema files into ::Lutaml::Uml::Document
+      class Xml
+        def self.load_schema(schema, root_schema)
+          result = Shale::Schema.from_xml([schema])
+
+          result.values.each do |klass|
+            # Temporary solution will update in the parser
+            klass = klass.gsub(/^require.*?\n/, "")
+            klass = klass.gsub(/< Shale::Mapper/, "< Lutaml::Xml::Mapper")
```

Comment:
We need to fork the gem

Reply:
@ronaldtse Sure, I'll fork this and open a PR in that repo.

Reply:
@ronaldtse Forked the gem at lutaml/shale -> https://github.com/lutaml/shale